terraform/azure_rke2_cluster/examples/all_supported_versions.tfvars

```diff
@@ -0,0 +1,32 @@
+nodes = [
+  {
+    name     = "linux-server"
+    image    = "linux"
+    roles    = ["etcd", "controlplane", "worker"]
+    replicas = 1
+  },
+  {
+    name     = "windows-worker"
+    image    = "windows-2022-core"
+    roles    = ["worker"]
+    replicas = 1
+  },
+  {
+    name = "windows-worker-2"
+    image = "windows-2019-core"
+    roles = ["worker"]
+    replicas = 1
+  },
+  {
+    name     = "windows-worker-3"
+    image    = "windows-2022"
+    roles    = ["worker"]
+    replicas = 1
+  },
+  {
+    name     = "windows-worker-4"
+    image    = "windows-2019"
+    roles    = ["worker"]
+    replicas = 1
+  }
```

Comment:
Why not name these `<image>-worker` instead? i.e .`windows-2022-core-worker`?

These would currently produce something like `windows-worker-4-0` with 1 replica, whereas it would be easier if it said `windows-2019-worker-0` instead.